src/Global/atoms/UserDrawerMeta.tsx

```diff
@@ -28,6 +28,7 @@ const LabelTitle = styled.div`
   color: #575757;
   font-family: ${({ theme }) => theme.fontFamily.ui};
   margin-top: 2px;
+  user-select: all;
```

Comment:
I think we do not need `user-select` property to select text, its by default selectable unless we made it un-selectable which happens in this case, where you first made `user-select: none`, then uses this `user-select: all` to select. I think you just need to remove this `user-select: none`.